modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/servlet/ServletContextUtil.java

```diff
@@ -101,7 +101,7 @@ public static InfoListRendererTracker getInfoListRendererTracker() {
 	}
 
 	public static LayoutDisplayPageProviderRegistry
-		getLayoutDisplayPageProviderTracker() {
+		getLayoutDisplayPageProviderRegistry() {
 
 		return _layoutDisplayPageProviderRegistry;
 	}
```

Comment:
@W2jx, please merge the current commit and the commit https://github.com/liferay-core-infra/liferay-portal/pull/2318/commits/a6c9d2c1be1fc594a1d9ff52d84a22155712f18d to the commit https://github.com/liferay-core-infra/liferay-portal/pull/2318/commits/0909d6c7bdbcf69e43b0bfef77be0d4075fece0e. We may treat these as apply usage. Otherwise, it look weird.